src/app/common/alert-list/alert-list.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AlertListComponent } from './alert-list.component';
+
+describe('NotFoundComponent', () => {
```

Comment:
This is my sample code, the name should not be NotFoundComponent, please check

Reply:
Yes, I have rename it now.